src/BaGet.Core/Configuration/BaGetOptions.cs

```diff
@@ -16,6 +16,10 @@ public class BaGetOptions
         /// </summary>
         public string PathBase { get; set; }
 
+        /// <summary>
+        /// The application Host without port number. If not set all hosts will be allowed. See https://github.com/aspnet/Announcements/issues/295
```

Comment:
Nice link! 👍

Reply:
Found it in your comments and it makes sense to explain why this is required.